server/src/firebase/model.ts

```diff
@@ -1,15 +1,22 @@
 import { ROOT_DIR } from '../const'
 
 console.log('***** Loading firestore database *****');
-var admin = require('firebase-admin');
+//TODO: determine if these changes are necessary
```

Comment:
Did you figure out if these changes are necessary?

Reply:
Yes they are, otherwise I cannot use firebase types (doesnt get admin as a namespace).

Reply:
Alright, we can remove the TODO then.